Source/sound.cpp

```diff
@@ -109,6 +122,19 @@ static int CapVolume(int volume)
 	return volume - volume % 100;
 }
 
+void ClearDuplicateSounds() {
+	duplicateSounds.clear();
+}
+
+void CleanupFinishedDuplicateSounds()
+{
+	duplicateSounds.erase(
+	    std::remove_if(duplicateSounds.begin(), duplicateSounds.end(), [](const std::unique_ptr<SoundSample> &sound) {

```

Comment:
Can't we use `auto` for lambda arguments now?

Reply:
No, because we don't really have C++17 due to still supporting Amiga (and also GCC 5 for now)

Reply:
Actually turns out `auto` lambda arguments are supported even in GCC-5 :smile: